### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by PeakXing**
Add item translation

### CODE REVIEW

**Comment by TartaricAcid** [comment]
**Oh, It's terrible**
@3TUSK

**Comment by 3TUSK** [comment]
No it is not bad. This translation actually looks fine.

However, two questions to the translator:
1. Are you sure that there is no existed translation that can be used for reference?
2. Why adding your name/id at the beginning of file? You don't have to do that - the same goes to all previous translators who added name  on their own - because you can always use commit history or _git log_ to find your contribution record.

@jaquardo Please don't merge this pull request - as I have some concerns on several entries and I want confirmation from translators.

**Comment by CannonFotter** [comment]
Please DO NOT use Google Translate to translate language files. It isn't fine...I think.

**Comment by 3TUSK** [comment]
Just notice that there is also #337 floating around. Calling @Meow-J @Mrkwtkr

File: resources/assets/storagedrawers/lang/zh_CN.lang [hunk 1]
```diff
@@ -28,45 +28,56 @@ item.storagedrawers.upgradeVoid.name=清空升级
...
-item.storagedrawers.upgradeRedstone.max.name=Redstone Max Upgrade
-item.storagedrawers.upgradeRedstone.max.description=Emits signal for most full slot.
-item.storagedrawers.upgradeRedstone.min.name=Redstone Min Upgrade
-item.storagedrawers.upgradeRedstone.min.description=Emits signal for least full slot.
+item.storagedrawers.upgradeRedstone.combined.name=红石信号升级
+item.storagedrawers.upgradeRedstone.combined.description=发出红石信号.
+item.storagedrawers.upgradeRedstone.max.name=红石信号升级Max
+item.storagedrawers.upgradeRedstone.max.description=抽屉有物品发出红石信号.
+item.storagedrawers.upgradeRedstone.min.name=红石信号升级Min
+item.storagedrawers.upgradeRedstone.min.description=抽屉装满时发出红石信号.
```
Comment (Krasjet) [inline resources/assets/storagedrawers/lang/zh_CN.lang:38]: nope

Comment (3TUSK) [reply resources/assets/storagedrawers/lang/zh_CN.lang:38]: `Emits signal for most full slot`
根据储物数量最多的格子输出红石信号
`Emits signal for least full slot`
根据储物数量最少的格子输出红石信号

Thoughts?

**Comment by jaquadro** [comment]
I completely missed this.  Since I obviously cannot vet any of these translations myself, thanks for working together to ensure a quality translation.

**Pull request closed without merging**